content/manager/bootstrapping.md

```diff
@@ -165,6 +165,153 @@ Services:
+    | aws_access_key                  | AWS       |             | Access key to use when accessing AWS API.                                                                                                                                                                                                                                |
+    | aws_secret_key                  | AWS       |             | Secret key to use when accessing AWS API.                                                                                                                                                                                                                                |
+    | openstack_username              | OpenStack |             | Username to use when accessing openstack API.                                                                                                                                                                                                                            |
+    | openstack_password              | OpenStack |             | Password to use when accessing openstack API.                                                                                                                                                                                                                            |
+    | openstack_auth_url              | OpenStack |             | Authentication URL to use when accessing openstack API.  e.g. http://myopenstack:5000/.                                                                                                                                                                                  |
+    | openstack_tenant_name           | OpenStack |             | Tenant name to use when accessing openstack API.                                                                                                                                                                                                                         |
+    | openstack_region                | OpenStack |             | Region to use when accessing openstack API.                                                                                                                                                                                                                              |
+    | manager_names_and_ips           |           | ✓           | You should include the manager's public IP here, as well as any DNS names you want to assign to the manager (comma separated). Internal IPs on your platform will be added automatically.  e.g. 192.0.2.54,mycloudify.example.com,mycloudify.dev.example.com,192.0.2.100 |
```

Comment:
Yes, but what /is/ this? (it's the names and IPs that will be added to the SSL certificate)